### DISCUSSION TITLE
Missing release notes from `protocolbuffers/protobuf`

### DISCUSSION THREAD

**Discussion by evg-tso** [renovatebot/renovate#22465] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

Since the 3.21.0 release, the versioning schema has changed: [read more here](https://developers.google.com/protocol-buffers/docs/news/2022-05-06).  

Since then, the release note from `3.21.0` are released as `21.0` (for example).  
Latest release notes: https://github.com/protocolbuffers/protobuf/releases/tag/v23.2.  
Example of a PR with missing release notes of the same version: https://github.com/evg-tso/sample-donkey-api/pull/284

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I don't think there's any easy way to support this

**Reply by evg-tso** [reply]
Perhaps you can substring away the major version starting with `3.21.0` when searching for the release notes?  
It's hacky, though.

**Reply by rarkins** [reply, marked as answer]
I prefer to avoid package-specific hacks. Been a while since I've seen a versioning I disliked as much as theirs
